frontend/src/components/annotation/annotation_ui_factory.vue

```diff
@@ -82,12 +82,13 @@
               @open_view_edit_panel="handle_open_view_edit_panel"
               @global_compound_attribute_change="on_global_compound_attribute_change"
 
-              @delete_text_instance="delete_text_instance"
               @hover_text_instance="hover_text_instance"
               @stop_hover_text_instance="stop_hover_text_instance"
-              @update_text_attribute="update_text_attribute"
-              @change_text_instance_lable="change_text_instance_lable"
-              @on_select_text_instance="on_select_text_instance"
+
+              @on_delete_instance="delete_instance"
```

Comment:
Instead of specific `delete_text_instance` use more generic `delete_instance` that can be used for different type of annotation intances. Do the same for `on_select_instance`, `on_change_intance_label`, and `on_updte_attribute`